docs/user/guides/build/webhooks.rst

```diff
@@ -1,46 +1,28 @@
-How to setup build notifications and webhooks
-=============================================
+How to setup build status webhooks
+==================================
 
-In this guide, you can learn how to setup a number of build notification mechanisms.
-Build notifications can alert you when your builds fail so you can take immediate action.
+In this guide,
+you can learn how to setup build notifications via webhooks.
 
-.. note::
+Read the Docs can notify external :term:`webhooks <webhook>` when builds are triggered, successful or failed.
+In that way,
+you can receive build notifications in your own monitoring channels and be alerted you when your builds fail so you can take immediate action.
```

Comment:
Isn't what we're doing a webhook? I don't think we'd "notify external webhooks", but instead we're "sending a webhook"?

Reply:
You are right. Wikipedia uses this definition:

> Webhooks are "user-defined HTTP callbacks".

Hoping this is the correct version: _We are notifying an external API using our webhooks._